src/cmake/init.py.in

```diff
@@ -0,0 +1,11 @@
+import os
+
+# Windows only
+if hasattr(os, 'add_dll_directory'):
+
+    paths = os.environ['path'].split(';')
+    paths.reverse()
```

Comment:
Could you add a comment to explain why we should reverse the order?

Reply:
Outdated, "reverse" has been removed